liquid_node/commands.py

```diff
@@ -108,13 +113,15 @@ def deploy():
 
     consul.set_kv('liquid_domain', config.liquid_domain)
     consul.set_kv('liquid_debug', 'true' if config.liquid_debug else 'false')
+    consul.set_kv('liquid_http', config.liquid_http)
```

Comment:
Let's name these `liquid_http_protocol`or something similarly specific. Maybe we could use a `liquid_public_url` entry in the consul kv instead of that `${liquid_http}://{{key "liquid_domain"}}` that keeps appearing in this patch.